src/styles/utils/_mixins.scss

```diff
@@ -0,0 +1,13 @@
+@mixin watch-size {
+  width: $watch-size;
+  height: $watch-size;
+}
+
+@mixin dot-size {
+  width: $dot-size;
+  height: $dot-size;
```

Comment:
Ok, but you also could create one mixin for circle (add border radius here) and use it for both watch and dot